cvat/apps/engine/serializers.py

```diff
@@ -185,14 +185,14 @@ class JobReadSerializer(serializers.ModelSerializer):
     project_id = serializers.ReadOnlyField(source="get_project_id", allow_null=True)
     start_frame = serializers.ReadOnlyField(source="segment.start_frame")
     stop_frame = serializers.ReadOnlyField(source="segment.stop_frame")
-    assignee = BasicUserSerializer(allow_null=True)
-    dimension = serializers.CharField(max_length=2, source='segment.task.dimension')
-    labels = LabelSerializer(many=True, source='get_labels')
+    assignee = BasicUserSerializer(allow_null=True, read_only=True)
+    dimension = serializers.CharField(max_length=2, source='segment.task.dimension', read_only=True)
```

Comment:
@zhiltsov-max , why do we need read_only=True here if below read_only_fields is specified?

Reply:
For some reason the fields marked with `read_only=True` are required otherwise, even though they are mentioned in `Meta.read_only_fields`.